packages/cli-main/src/cli/commands/upgrade.ts

```diff
@@ -7,6 +7,7 @@ import {
   getPackageManager,
 } from '@shopify/cli-kit/node/node-package-manager'
 import Command from '@shopify/cli-kit/node/base-command'
+import {getCliProjectDir} from '@shopify/cli-kit/src/cli.js'
```

Comment:
This import is invalid and will break in production. You'll have to use the exported modules instead:

```ts
import { cli } from "@shopify/cli-kit"
```